docs/src/3.4.MouseEvents.mdx

```diff
@@ -4,13 +4,14 @@ import { MouseEvents, MouseEventsInstanced } from './jsx/allLiveEditors';
 
-| Name            | type           | Default | Description                                                                                  |
-| --------------- | -------------- | ------- | -------------------------------------------------------------------------------------------- |
-| `onDoubleClick` | `MouseHandler` |         |                                                                                              |
-| `onMouseDown`   | `MouseHandler` |         |                                                                                              |
-| `onMouseUp`     | `MouseHandler` |         |                                                                                              |
-| `onMouseMove`   | `MouseHandler` |         | only available if the `hitmapOnMouseMove` prop is set to `true` on the Worldview component |
-| `onClick`       | `MouseHandler` |         |                                                                                              |
+| Name            | type              | Default | Description                                                                                                                                                              |
+| --------------- | ----------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `onDoubleClick` | `MouseHandler`    |         |                                                                                                                                                                          |
+| `onMouseDown`   | `MouseHandler`    |         |                                                                                                                                                                          |
+| `onMouseUp`     | `MouseHandler`    |         |                                                                                                                                                                          |
+| `onMouseMove`   | `MouseHandler`    |         | only available if the `hitmapOnMouseMove` prop is set to `true` on the Worldview component                                                                               |
+| `onClick`       | `MouseHandler`    |         |                                                                                                                                                                          |
+| `getHitmap`     | `AssignNextIdsFn` |         | Maps the `children` to props to be drawn on the hitmap. Optional, but required for any mouse interactions. Provided by default by all included Commands. See more below. |
```

Comment:
📝 Naming: usually in our terminology the "hitmap" is the whole offscreen canvas, so calling this "getHitmap" is a bit weird. Maybe `getChildrenForHitmap` or something that relates it more to the original object/children?

Reply:
I think that makes sense